src/backend/columnar/Makefile

```diff
@@ -0,0 +1,22 @@
+citus_subdir = src/backend/columnar
+safestringlib_srcdir = $(citus_abs_top_srcdir)/vendor/safestringlib
+SUBDIRS = . safeclib
+SUBDIRS +=
+OBJS += \
+	$(patsubst $(citus_subdir)/%.c,%.o,$(foreach dir,$(SUBDIRS), $(sort $(wildcard $(citus_subdir)/$(dir)/*.c))))
```

Comment:
Use `$(citus_abs_srcdir)` instead of `$(citus_subdir)` in both places in this line.

Reply:
done